build.gradle

```diff
@@ -58,8 +59,10 @@ repositories {
 }
 
 dependencies {
+    implementation "gradle.plugin.net.wooga.gradle:atlas-dotnet-sonarqube:0.0.2"
+    implementation "gradle.plugin.net.wooga.gradle:atlas-unity:2.2.0-rc.1"
+    implementation 'com.wooga.gradle:gradle-commons:[0, 1)'
```

Comment:
please don't follow the `0.x` version. Bind it to `0.1.0` or at least `(0.1,0.2]`